src/sql/media/icons/common-icons.css

```diff
@@ -652,3 +652,35 @@ Includes non-masked style declarations. */
 	width: 32px;
 	height: 32px;
 }
+
+/** Info icons **/
+
+.dialog-message.info .dialog-message-icon {
```

Comment:
Remove any specific classes like `dialog-message` or `modelview-text-tooltip`. This stylesheet is supposed to be generic for the entire app so shouldn't be doing anything specifically for parts of it.

and there should only be two definitions here, one for dark and one for normal color themes.

Reply:
I was just trying to bring over similar classes that used the same icon